py3-astroid.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: py3-astroid
+  version: 3.0.1
+  epoch: 0
+  description: A common base representation of python source code for pylint and other projects
+  copyright:
+    - license: LGPL-2.1-or-later
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-pip
+      - py3-setuptools
+      - python3
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/pylint-dev/astroid
+      tag: v${{package.version}}
+      expected-commit: 0191f534213364a844e8138dd68bb71f3ba603e3
+
+  - name: Python Build
+    runs: python -m pip install --prefix=/usr --root="${{targets.destdir}}" .
```

Comment:
I don't think we have any firm docs on packaging python but from the recent python 3.12 upgrade I think there was a suggestion that `pip install` wasn't great as it can vendor other python dependencies.  We now have some reusable melange pipelines that might be better https://github.com/chainguard-dev/melange/tree/main/pkg/build/pipelines/python

Reply:
Ah, 🤦 ! I think I even knew that. I blindly followed the preferred build steps in the `astroid` README.

The new commit uses an approach other than `pip install`.

In your opinion, what docs should I update with this advice? The internal wiki? Somewhere else?

Reply:
THANK YOU, BTW.